### ISSUE TITLE
Target class [subscription] does not exist.

### ISSUE THREAD

**Issue by anasalsaadi14** [imdhemy/laravel-in-app-purchases#110]
When handled and go ahead to get details of. subscription, in Catch block
```

<?php

namespace App\Listeners;

use App\Models\Subscription;
use Imdhemy\Purchases\Events\GooglePlay\SubscriptionRenewed as SubscriptionRenewedEvent;
use Illuminate\Contracts\Queue\ShouldQueue;
use Illuminate\Queue\InteractsWithQueue;

class SubscriptionRenewed
{
    /**
     * Create the event listener.
     *
     * @return void
     */
    public function __construct()
    {

    }

    public function handle(SubscriptionRenewedEvent $event)
    {
     
        try {

            $SubscriptionIdentifier = $event->getSubscriptionIdentifier();
            $notification = $event->getServerNotification();
            $SubscriptionId = $event->getSubscriptionId();
            $type = $notification->getType();
            $bundle = $notification->getBundle();
            $subscription = $notification->getSubscription();
            $uniqueIdentifier = $subscription->getUniqueIdentifier();
            $itemId = $subscription->getItemId();
            $expirationTime = $subscription->getExpiryTime();
            $provider = $subscription->getProvider();
         
        } catch (\Exception $e) {

     echo $e->getMessage();
        }

    }
}

```

**Comment by khizar1556** [comment]
Go to this file **config/app.php**
Add this line it will solve your problem
`'subscription' => Imdhemy\Purchases\Subscription::class`

**Comment by imdhemy** [comment]
Fixed in #117
